dag/demography.yml

```diff
@@ -156,3 +156,18 @@ steps:
     - data://garden/countries/2023-09-25/gleditsch
   data://grapher/demography/2023-12-20/population_fariss:
     - data://garden/demography/2023-12-20/population_fariss
+
+  # HYDE 3.3
+  data://meadow/hyde/2024-01-03/general_files:
```

Comment:
Also, maybe it would have been more convenient to have just one meadow step, with two tables, and possibly just one garden step, also with two tables.